docs/public/replay-viewer-app/style.css

```
@@ -754,6 +754,21 @@ button:disabled {
   box-shadow: 0 0 12px rgb(202 166 255 / 40%);
 }
 
+.legend-swatch--connector-ingest {
```

Stylelint (17.13.0)

[error] Expected class selector ".legend-swatch--connector-ingest" to be kebab-case (selector-class-pattern)

(selector-class-pattern)